lambdas/package.json

```diff
@@ -1,50 +1,51 @@
 {
-  "name": "di-ipv-common-lambdas",
-  "version": "1.0.0",
-  "description": "Common node Lambdas",
-  "main": "app.js",
-  "dependencies": {
-    "@aws-lambda-powertools/logger": "1.7.0",
-    "@aws-lambda-powertools/metrics": "1.7.0",
-    "@aws-lambda-powertools/tracer": "1.7.0",
-    "@aws-sdk/client-dynamodb": "3.363.0",
-    "@aws-sdk/client-kms": "3.363.0",
-    "@aws-sdk/client-sqs": "3.363.0",
-    "@aws-sdk/client-ssm": "^3.363.0",
-    "@aws-sdk/credential-providers": "3.363.0",
-    "@aws-sdk/lib-dynamodb": "3.150.0",
-    "@middy/core": "^4.2.8",
-    "esbuild": "0.17.15",
-    "jose": "^4.13.1"
-  },
-  "scripts": {
-    "unit": "jest --config=jest.config.ts",
-    "lint:prettier": "prettier --check src tests",
-    "lint:eslint": "eslint .",
-    "lint": "npm run lint:prettier && npm run lint:eslint",
-    "compile": "tsc",
-    "test": "npm run compile && npm run unit"
-  },
-  "devDependencies": {
-    "@types/aws-lambda": "8.10.114",
-    "@types/jest": "29.5.0",
-    "@types/node": "18.15.11",
-    "@typescript-eslint/eslint-plugin": "5.10.2",
-    "@typescript-eslint/parser": "5.10.2",
-    "aws-sdk": "2.1351.0",
-    "esbuild-jest": "0.5.0",
-    "eslint": "8.8.0",
-    "eslint-config-prettier": "8.3.0",
-    "eslint-plugin-prettier": "4.0.0",
-    "jest": "29.5.0",
-    "prettier": "2.5.1",
-    "ts-jest": "29.1.0",
-    "ts-node": "10.9.1",
-    "typescript": "5.0.3"
-  },
-  "keywords": [
-    "access-token"
-  ],
-  "author": "",
-  "license": "ISC"
+    "name": "di-ipv-common-lambdas",
+    "version": "1.0.0",
+    "description": "Common node Lambdas",
+    "main": "app.js",
+    "dependencies": {
+        "@aws-lambda-powertools/logger": "1.7.0",
+        "@aws-lambda-powertools/metrics": "1.7.0",
+        "@aws-lambda-powertools/tracer": "1.7.0",
+        "@aws-lambda-powertools/parameters": "1.17.0",
+        "@aws-sdk/client-dynamodb": "3.363.0",
+        "@aws-sdk/client-kms": "3.363.0",
+        "@aws-sdk/client-sqs": "3.363.0",
+        "@aws-sdk/client-ssm": "^3.363.0",
```

Comment:
Should peg this to just like the ones above it, don't want to upgrading while the others are the same version